### DISCUSSION TITLE
[QIITA CLI] npx qiita pull で限定共有記事を読み込めていない？

### DISCUSSION THREAD

**Discussion by zacky1972** [increments/qiita-discussions#585] [要望・アイデア・不具合]
表題のとおり，npx qiita pull で限定共有記事を読み込めていないようですので，限定共有記事を取り込むオプションとかありましたらご教示ください．無ければ機能追加要望します．

**Comment by tanaka-asami415** [comment]
@zacky1972 
Qiita CLIをご利用いただきありがとうございます！

`includePrivate:` で限定共有記事を含めるかどうかを選べるのですが、
デフォルトが`false` になっているため`true` にしていただくと読み込めると思います！

詳細は下記をご参照ください🙇
[ユーザー設定ファイルについて](https://github.com/increments/qiita-cli?tab=readme-ov-file#%E3%83%A6%E3%83%BC%E3%82%B6%E3%83%BC%E8%A8%AD%E5%AE%9A%E3%83%95%E3%82%A1%E3%82%A4%E3%83%AB%E3%81%AB%E3%81%A4%E3%81%84%E3%81%A6)

**Reply by zacky1972** [reply]
ありがとうございます！
